tutorials/notebooks/color-excess/color-excess.ipynb

```diff
@@ -0,0 +1,866 @@
+    "\n",
+    "\n",
+    "\n",
+    "## Summary\n",
+    "\n",
+    "In this tutorial, we will look at some extinction curves from the literature, use one of those curves to deredden an observed spectrum, and practice invoking a background source flux in order to calculate magnitudes from an extinction model.\n",
+    "\n",
+    "The primary libraries we'll be using are [dust_extinction](https://dust-extinction.readthedocs.io/en/latest/) and [synphot](https://synphot.readthedocs.io/en/latest/), which are [Astropy affiliated packages](https://www.astropy.org/affiliated/). \n",
+    "\n",
+    "We recommend installing the two packages in this fashion:\n",
+    "```\n",
+    "conda install -c http://ssb.stsci.edu/astroconda synphot\n",
+    "pip install dust_extinction\n",
+    "```\n",
+    "\n",
+    "Notice that the structure of `dust_extinction` changed significantly from v0.6 to v0.7, so updating your installation may be required. If so, you can upgrade the package with pip:\n",
+    "```\n",
+    "pip install --upgrade dust_extinction\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "vFDq1xGXz_t4"
+   },
+   "outputs": [],
+   "source": [
+    "import matplotlib.pyplot as plt\n",
+    "%matplotlib inline\n",
+    "\n",
+    "import numpy as np\n",
+    "import astropy.units as u\n",
+    "from astropy.table import Table\n",
+    "from dust_extinction.parameter_averages import CCM89, F99\n",
+    "from synphot import units, config\n",
+    "from synphot import SourceSpectrum,SpectralElement,Observation,ExtinctionModel1D\n",
+    "from synphot.models import BlackBodyNorm1D\n",
+    "from synphot.spectrum import BaseUnitlessSpectrum\n",
+    "from synphot.reddening import ExtinctionCurve\n",
+    "from astroquery.simbad import Simbad\n",
+    "from astroquery.mast import Observations\n",
+    "import astropy.visualization"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "ssir1UjtD_Wn"
+   },
+   "source": [
+    "# Introduction"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "1rvwpEs946ju"
+   },
+   "source": [
+    "Dust in the interstellar medium (ISM) extinguishes background starlight.  The wavelength dependence of the extinction is such that short-wavelength light is extinguished more than long-wavelength light, and we call this effect *reddening*.\n",
+    "\n",
+    "If you're new to extinction, here is a brief introduction to the types of quantities involved.\n",
+    "The fractional change to the flux of starlight is \n",
+    "$$\n",
+    "\\frac{dF_\\lambda}{F_\\lambda} = -\\tau_\\lambda\n",
+    "$$\n",
+    "where $\tau$ is the optical depth and depends on wavelength.  Integrating along the line of sight, the resultant flux is an exponential function of optical depth,\n",
```

Comment:
This was a tricky one.  ``\t`` is interpreted as *tab* rather than "backslash-tab" in the raw file.  I dunno how that came to be, but if you edit in the notebook iterface it can either manually be changed to \t and does the right thing, or editing the raw file like I'm suggesting here seems to do the same thing.

```suggestion
    "where $\\tau$ is the optical depth and depends on wavelength.  Integrating along the line of sight, the resultant flux is an exponential function of optical depth,\n",
```

Reply:
I know. This was leftover from my fiddling with the syntax. I didn't bother reverting.